concourse/steps/component_descriptor.mako

```diff
@@ -165,6 +170,26 @@ component_v2.resources.append(
 %   endfor
 % endfor
 
+# add helmcharts (from publish-trait)
+% for helmchart in helmcharts:
+<%
+name = helmchart.name
+target_ref_prefix = f'{helmchart.registry}/{name}'
+%>
+component_v2.resources.append(
+  cm.Resource(
+    name='${helmchart.name}',
+    version=effective_version, # always inherited from component
+    type='helmchart',
```

Comment:
Why not re-using `cm.ResourceType.HELM_CHART` (`helmChart/v1`)?

Reply:
valid ask. I used the same type-str as used by ocm-cli if asking it to package a helmchart (ocm resource add). Although, according to [source](https://github.com/open-component-model/ocm/blob/main/pkg/contexts/ocm/resourcetypes/const.go#L16), I would actually expect OCM-CLI to acually use `helmChart` as typename.

@mandelsoft | @fabianburth : Could you shed some light into which resource-typename we should use for helmcharts so we are consistent w/ ocm-cli?

@8R0WNI3 : I suggest to change to helmChart for now, and change later, depending on Uwe/Fabian's responses.